installing/installing-mirroring-installation-images.adoc

```diff
@@ -32,6 +32,11 @@ If you have an entitlement to Red Hat Quay, see the documentation on deploying R
 
 * If you do not already have an existing solution for a container image registry, subscribers of {product-title} are provided a xref:../installing/installing-mirroring-installation-images.adoc#mirror-registry[mirror registry for Red Hat OpenShift]. The _mirror registry for Red Hat OpenShift_ is included with your subscription and is a small-scale container registry that can be used to mirror the required container images of {product-title} in disconnected installations.
 
+[NOTE]
+====
+Red Hat does not support and test third party registries with {product-title}. 
```

Comment:
If we mention RH doesn't support those registries, it would make users confused because we do list them in the doc. Imho, i'd suggest to only let users know that we don't test those third party registries listed in the doc.

Reply:
Addressed